### PR TITLE
Fix barbarian localizations

### PR DESCRIPTION
**Pull request by TikaelSol**

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -1165,7 +1165,7 @@
                     "Wolf": "Wolf"
                 },
                 "Dragon": {
-                    "NormalRageDamage": "Deal normal rage damage",
+                    "ToggleLabel": "Dragon Instinct Rage",
```
Comment (stwlam) [inline static/lang/re-en.json:1168]:
```suggestion
                    "ToggleLabel": "Draconic Rage",
```
After the class feature

File: packs/data/classfeatures.db/dragon-instinct.json [hunk 2]
```diff
@@ -182,7 +182,7 @@
             {
                 "domain": "all",
                 "key": "RollOption",
-                "label": "PF2E.SpecificRule.Instinct.ToggleProperty",
```
Comment (simonward) [inline packs/data/classfeatures.db/dragon-instinct.json:185]: "PF2E.SpecificRule.Barbarian.Dragon.ToggleLabel"

File: packs/data/classfeatures.db/dragon-instinct.json [hunk 3]
```diff
@@ -182,7 +182,7 @@
...
+                "label": "PF2E.SpecificRule.Dragon.ToggleLabel",
```
Comment (simonward) [inline packs/data/classfeatures.db/dragon-instinct.json:185]: "PF2E.SpecificRule.Barbarian.Dragon.ToggleLabel"

Comment (TikaelSol) [inline packs/data/classfeatures.db/dragon-instinct.json:185]:
```suggestion
                "label": "PF2E.SpecificRule.Barbarian.Dragon.ToggleLabel",
```

File: static/lang/re-en.json [hunk 4]
```diff
@@ -1179,6 +1179,9 @@
                     "NegativeDamage": "Deal Negative Damage",
                     "PositiveDamage": "Deal Positive Damage",
                     "ToggleLabel": "Spirit Instinct Rage"
```
Comment (stwlam) [inline static/lang/re-en.json:1181]:
```suggestion
                    "ToggleLabel": "Spirit Rage"
```

File: static/lang/re-en.json [hunk 5]
```diff
@@ -1179,6 +1179,9 @@
                     "NegativeDamage": "Deal Negative Damage",
                     "PositiveDamage": "Deal Positive Damage",
```
Comment (stwlam) [inline static/lang/re-en.json:1180]:
```suggestion
                    "NegativeDamage": "Negative Damage",
                    "PositiveDamage": "Positive Damage",
```

**Pull request merged**